vivisect/__init__.py

```diff
@@ -1274,6 +1274,20 @@ def makeOpcode(self, va, op=None, arch=envi.ARCH_DEFAULT):
 
         return loc
 
+    def reprLocEntry(self, va):
```

Comment:
it's not strictly necessary.  it was something i wanted for troubleshooting.  i wrote it before i remembered reprLocation existed... in fact, i named it just that :)  then renamed to avoid name collision.  
it's purpose is different than reprLocation, and i'm ok with some other convention.

```
In [9]: vw.reprLocation(vw.getLocation(0x1613c-1))
Out[9]: 'andeq r10, r3, r8 , lsr #1'

In [10]: vw.reprLocEntry(0x1613c-1)
Out[10]: '(0x16138, 4, LOC_OP, 196640)  # Opcode'
```

Reply:
`_dbgLocEntry()` ok?

Reply:
:+1: